AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -140,7 +140,13 @@ public class CardComponent: PresentableComponent,
         
         cardViewController.stopLoading()
     }
-    
+
+    public func submit() {
+        guard !configuration.showSubmitButton else { return }
```

Comment:
I decided to prevent any kind of submit action if the default submit button is enabled. What do you think?

Reply:
Make sense :)